packages/itwinui-css/backstop/tests/input-grid.html

```diff
@@ -0,0 +1,378 @@
+        <div class="iui-status-message">
+          <div>This is message</div>
+        </div>
+      </div>
+
+      <div
+        class="iui-input-grid"
+        data-iui-label-placement="inline"
+      >
+        <label
+          class="iui-input-label"
+          data-iui-disabled="true"
+          for="input-inline-03"
+        >This is label</label>
+        <input
+          class="iui-input"
+          disabled
+          id="input-inline-03"
+          placeholder="Disabled input..."
+        />
+        <div class="iui-status-message">
+          <div>This is message</div>
+        </div>
+      </div>
+
+      <div
+        class="iui-input-grid"
+        data-iui-label-placement="inline"
+      >
+        <label
+          class="iui-input-label"
+          data-iui-disabled="true"
+          for="input-inline-04"
+        >This is label</label>
+        <input
+          class="iui-input"
+          disabled
+          id="input-inline-04"
+          placeholder="Input..."
+          value="Disabled value"
+        />
+        <div class="iui-status-message">
+          <div>This is message</div>
+        </div>
+      </div>
+
+      <div
+        class="iui-input-grid"
+        data-iui-label-placement="inline"
+      >
+        <label
+          class="iui-input-label iui-required"
+          for="input-inline-05"
+        >This is required label</label>
+        <input
+          class="iui-input"
+          required
+          id="input-inline-05"
+          placeholder="Required input..."
+        />
+        <div class="iui-status-message">
+          <div>This is message</div>
+        </div>
+      </div>
+
+      <div
+        class="iui-input-grid"
+        data-iui-label-placement="inline"
+      >
+        <label
+          class="iui-input-label"
+          for="input-inline-06"
+        >This is positive label</label>
+        <input
+          class="iui-input"
+          id="input-inline-06"
+          placeholder="Positive input..."
+          data-iui-status="positive"
+        />
+        <div
+          class="iui-status-message"
+          data-iui-status="positive"
+        >
+          <div>This is message</div>
+        </div>
+      </div>
+
+      <div
+        class="iui-input-grid"
+        data-iui-label-placement="inline"
+      >
+        <label
+          class="iui-input-label"
+          for="input-inline-07"
+        >This is warning label</label>
+        <input
+          class="iui-input"
+          id="input-inline-07"
+          placeholder="Warning input..."
+          data-iui-status="warning"
+        />
+        <div
+          class="iui-status-message"
+          data-iui-status="warning"
+        >
+          <div>This is message</div>
+        </div>
+      </div>
+
+      <div
+        class="iui-input-grid"
+        data-iui-label-placement="inline"
+      >
+        <label
+          class="iui-input-label"
+          for="input-inline-08"
+        >This is negative label</label>
```

Comment:
maybe make all label texts the same so that we can vertical alignment? 
<img width="209" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/97c4120d-8088-46d6-b814-e9b34573ea06">

Reply:
updated labels